packages/common/tests/crypto/hex2abc.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai';
+
+import { hex2abc } from '@/crypto/hex2abc';
+
+describe('hex2abc()', () => {
```

Comment:
I just realized we don't have a test that includes values outside of the mapping. It looks like those are dropped, is that what we want @jonahsnider?

Reply:
the resulting string will contain `undefined` for each char outside the expected range.
i think that's ok as i don't expect we'll be using this function outside of crypto utils that are guaranteed to be given a valid hex string